src/cargo.md

```diff
@@ -5,6 +5,26 @@ used in the Rust ecosystem to build and run Rust applications. Here we want to
 give a brief overview of what Cargo is and how it fits into the wider ecosystem
 and how it fits into this training.
 
+## Installation
+
+### Rustup (Recommended)
+
+You can follow the instructions to install cargo and rust compiler, among other standard ecosystem tools with the [rustup][3] tool, which is maintained by the Rust Foundation.
+
+On Unix-like systems you can curl down the install script and pipe it into your shell like so:
+
+```shell
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

Comment:
I would like us to not add this line of magic... I prefer it if there is one place on the Internet with the recommended command. Put differently: we should absolutely mention and recommend `rustup.rs`, but we should let them explain how to install in details.

Reply:
You are absolutely right, this is subject to change, but a link will always be right